Add persistence to MariaDB chart

viglesiasce opened the pull request.

krancour commented:
cc @rimusz since this is a refactor of one of your contributions.

Commit:
Move first batch of PVC charts to stable

Commit:
Add notes for mariadb

Commit:
Add persistence to MariaDB chart

Commit:
Add persistence to MariaDB chart

prydonius commented on stable/mariadb/templates/pvc.yaml:
```diff
@@ -0,0 +1,15 @@
+{{- if .Values.persistence.enabled }}
+---
```
Can we remove this? Helm already inserts these separators.

prydonius commented on stable/mariadb/values.yaml:
```diff
@@ -25,3 +25,10 @@ imageTag: 10.1.14-r3
+
+## Persist data to a persitent volume
+persistence:
+    enabled: true
```
other YAML files use 2 spaces for indentation, can we change this to 2 spaces too?

prydonius commented:
closes #67

prydonius commented:
We should also update the README to add the new values

prydonius commented on stable/mariadb/templates/pvc.yaml:
```diff
@@ -0,0 +1,15 @@
+metadata:
+  name: {{ template "fullname" . }}
+  annotations:
+    volume.alpha.kubernetes.io/storage-class: {{ .Values.persistence.storageClass | quote }}
```
Should this be `volume.beta` now? Or are we keeping the alpha label to support 1.3?

viglesiasce replied:
Unfortunately the annotation did not change but the support did.

Check out the following PR to the docs:
https://github.com/kubernetes/kubernetes.github.io/pull/1311/files#diff-e3ed4a5817ba0330edba5ec941757bd0R416

Commit:
Add persistence values to README, fix indent

Commit:
Add persistence values to README, fix indent

viglesiasce commented:
All updates are in per review.

Commit:
Update command for connecting to mariadb

prydonius committed:
Merge pull request #89 from viglesiasce/mariadb-notes

Add notes for mariadb

Commit:
Handle moves in CI (#96)

prydonius commented on incubator/mysql/templates/pvc.yaml:
```diff
@@ -1,5 +1,4 @@
 {{- if .Values.persistence.enabled }}
----
```
looks like you meant to do this in `stable/mariadb/templates/pvc.yaml`

viglesiasce replied:
indeed!!!!

Commit:
Move mysql to stable, add NOTES.txt (#91)

* Move mysql to stable, add NOTES

* Add note for password

* Formatting issues for mysql chart

Commit:
Remove markup from mariadb pvc

prydonius committed:
mariadb 0.3.1 (#92)

Updates MariaDB image version to 10.1.17-r1 including fixes for the
OpenSSL security update
(https://www.openssl.org/news/secadv/20160922.txt)

Commit:
Fix README conflict

Commit:
Bump mariadb to version 0.4.0

Commit:
Fix merge conflict

viglesiasce commented:
Will reopen another branch to clean up some of the madness in the commits here.
